conversion.php

```diff
@@ -1,5 +1,78 @@
 <?php
 
+        function fahrCel($temp) // converts Fahrenheit to Celcius and rounds the conversion to two decimals
```

Comment:
What do these functions do that the previous lines of code didn't? If I'm reading the addition tracker correctly we're adding some lines here. I feel like it's possible to reduce the file size with functions. 

I'm seeing that these functions have a few things in common. They all take in a temperature value. They all output a rounded number. They all do some calculation involving a combination of 32, 0.55, 1.8, and/or 459.67.

Since we're conditionally calling the functions anyway, based on the name of the form input, I wonder if everything could be bundled into one function that takes two arguments. Like:

`function convertThis($type, $temp){
  if ($type == "fahr") {
    $out = ((int) $degree - 32) * (5/9);
  } else if ($type == "cel"){}
  ...
}`

And so on. 

Do you think that's a good idea?

Reply:
That's a great idea! I had the same thought about length. I don't have the time available to do more on this today so feel free to go ahead if you do. Otherwise I'll take a look tomorrow. Thanks, Michael! Nice job spotting the pattern.

Reply:
Although I do think it might need to take three arguments? Degrees, original type, new type.